src/shared/api/http-client.ts

```
@@ -0,0 +1,221 @@
+
+type CancelToken = Symbol | string | number;
+
+export enum ContentType {
+  Json = 'application/json',
+  FormData = 'multipart/form-data',
+  UrlEncoded = 'application/x-www-form-urlencoded',
+  Text = 'text/plain',
+}
+
+export class HttpClient<SecurityDataType = unknown> {
+  public baseUrl: string = 'https://dev.api.cchaksa.com';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)